scripts/output_env.sh

```diff
@@ -36,7 +36,7 @@ print_version()
     fi
 
     BIN=`which "$BIN"`
-    VERSION_STR=`$BIN $ARGS 2>&1`
+    VERSION_STR=`$BIN $ARGS 2>/dev/null`
```

Comment:
This is a problem for getting the python version. `python --version` outputs to `stderr` until python 3.4, so this script won't work for versions earlier than that.

Reply:
That's not what I was expecting! I've reverted that change and adapted the script. Thanks for the feedback!